src/main/java/core/jdbc/JdbcTemplate.java

```diff
@@ -0,0 +1,62 @@
+package core.jdbc;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JdbcTemplate {
+    private static final JdbcTemplate jdbcTemplate = new JdbcTemplate();
+
+    public static JdbcTemplate getInstance() {
+        return jdbcTemplate;
+    }
+
+    public void update(String sql, Object... parameters) {
+        try (PreparedStatement pstmt = toPreparedStatementByObjects(sql, parameters)) {
+            pstmt.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataAccessException(e);
+        }
+    }
+
+    public <T> T queryForObject(String sql, RowMapper<T> rowMapper, Object... parameters) {
+        try (PreparedStatement pstmt = toPreparedStatementByObjects(sql, parameters);
+             ResultSet rs = pstmt.executeQuery()) {
+
+            List<T> result = queryForList(sql, rowMapper, parameters);
+
+            if (result.isEmpty()) {
+                return null;
+            }
+            return result.get(0);
+
+        } catch (SQLException e) {
+            throw new DataAccessException(e);
+        }
```

Comment:
```suggestion
        List<T> result = queryForList(sql, rowMapper, parameters);
```

이 메서드를 호출하면 try/catch는 없어도 되지 않을까?